uint16 numberOfScans
uint16 speed
uint16 holdUpTime
uint16 startPosition
uint16 endPosition

